ibis-server/Makefile

```
@@ -13,3 +13,9 @@ dev:
 
 test:
 	poetry run pytest
+
+docker-build:
+	docker image build . -t wren-engine-ibis
+
+docker-run:
```

```suggestion
dev-docker-run:
```
How about add a `dev` prefix? I guess someone maybe confuse on which version would be started by this command. The image on repo? or The local build.
I guess this one will start the local one, right?